build_debian.sh

```diff
@@ -312,9 +312,13 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     python-pip              \
     python3-pip             \
     cron                    \
-    haveged
-
+    haveged                 
 
+## Secure boot signed shim grub 
```

Comment:
>Secure boot signed shim grub  [](start = 3, length = 29)

The comment is difficult to read. Could you check against description of installed packages and refine the statement?